Alpaca.Markets/Interfaces/IHistoricalQuoteV2.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Alpaca.Markets
+{
+    /// <summary>
+    /// Encapsulates historical trade information from Polygon REST API.
+    /// </summary>
+    public interface IHistoricalQuoteV2
```

Comment:
I'm sure we can extend the existing `IHistoricalQuote` interface instead of creating a new one. Maybe I'll apply these changes in separate branch after merging this PR and send results for your review.